tsdb/db.go

```diff
@@ -54,8 +54,6 @@ const (
 	// Block dir suffixes to make deletion and creation operations atomic.
 	// We decided to do suffixes instead of creating meta.json as last (or delete as first) one,
 	// because in error case you still can recover meta.json from the block content within local TSDB dir.
-	// TODO(bwplotka): TSDB can end up with various .tmp files (e.g meta.json.tmp, WAL or segment tmp file. Think
-	// about removing those too on start to save space. Currently only blocks tmp dirs are removed.
```

Comment:
I would not remove this comment because we are only checking for tmp stuff inside wal and not everything mentioned in this comment.